libcloud/compute/types.py

```diff
@@ -222,25 +222,31 @@ class NodeState(Type):
     Standard states for a node
 
     :cvar RUNNING: Node is running.
+    :cvar STARTING: Node is starting up.
     :cvar REBOOTING: Node is rebooting.
     :cvar TERMINATED: Node is terminated. This node can't be started later on.
+    :cvar STOPPING: Node is currently trying to stop.
```

Comment:
I was up in the air on it too, thought about maybe using PENDING for a similar effect. It won't affect my waiting for state which is what this change is trying to bring. I was just trying to be as specific as possible on node states.
I'll wait for consensus from @tonybaloney